docs/git-guide.md

```diff
@@ -784,7 +784,7 @@ complicated rebase.
 When you're ready for feedback, submit a pull request. At Zulip we recommend
 submitting work-in-progress pull requests early and often. This allows you to
 get feedback and help with your bugfix or feature. Prefix work-in-progress pull
-requests with **[WIP]**.
+requests with **[WIP]**: this will indicate to maintainers that your pull request is not yet ready to be reviewed and merged.
```

Comment:
A WIP PR is not necessarily "not yet ready to be reviewed," but it definitely means "not yet ready to be merged."

A WIP helps the PR author to get initial feedback on what they are doing and if the path they are taking is the correct path to take.